spec/inquisition/factory_bot/runner_spec.rb

```diff
@@ -0,0 +1,36 @@
+RSpec.describe Inquisition::FactoryBot::Runner do
+  describe '#call' do
+    let(:runner) { described_class.new }
+    let(:linter) { instance_double(Inquisition::FactoryBot::FactoryBotModifiedLinter) }
```

Comment:
Do we need all these lets?
Can we do something like this:

```ruby
let(:linter) { instance_double(Inquisition::FactoryBot::FactoryBotModifiedLinter) }
let(:factory_error) { instance_double(FactoryBot .... FactoryError }
let(:factory_trait_error) { instance_double(FactoryBot .... FactoryTraitError }

before do
  allow(linter).to receive(....).and_return(
    [factory_error, factory_trait_error]
  )
end

it '...' do
  expect(runner.call).to contain_exactly(...)
end
```

what do you think?